frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -314,6 +320,7 @@ function Resource({
                   openInNewTab={openInNewTab}
                   url={href}
                   buyButtonText={buyButtonText}
+                  margin="0 0 0 12px"
```

Comment:
Same thing here, if I'm correct.
```suggestion
                  buttonStyling={{ ml: "12px" }}
```